### ISSUE TITLE
Missing alternative text : Image alternative text is not present

### ISSUE THREAD

**Issue by r-badani** [voidcosmos/angular-communities#25]
<img width="359" alt="missing-alt-text-a11y-error" src="https://user-images.githubusercontent.com/56524185/80854833-3b8e5780-8c09-11ea-8491-102e3fd43780.png">

**Why It Matters**
Each image must have an alt attribute. Without alternative text, the content of an image will not be available to screen reader users or when the image is unavailable.

**How to Fix It**
Add an alt attribute to the image. The attribute value should accurately and succinctly present the content and function of the image. If the content of the image is conveyed in the context or surroundings of the image, or if the image does not convey content or have a function, it should be given empty/null alternative text (alt="").

**Comment by r-badani** [comment]
I am working on it, will open a PR shortly.
